include/swift/Sema/CSFix.h

```diff
@@ -322,6 +322,11 @@ enum class FixKind : uint8_t {
   /// even though result type of the reference doesn't conform
   /// to an expected protocol.
   AllowInvalidStaticMemberRefOnProtocolMetatype,
+    
+  /// Allow the wrappedValue type of the outermost property
```

Comment:
If I understand code correctly - this fix would actually be applied for every composition, not just the outermost one. I think it would be great if comment did capture that.